setup.py

```diff
@@ -48,7 +48,7 @@ def package_files(directory):
     "entrypoints",
     "gitpython>=2.1.0",
     "pyyaml>=5.1",
-    "protobuf>=3.7.0",
+    "protobuf>=3.7.0,<4",
```

Comment:
MLflow 1.25.x is incompatible with `protobuf>=4.0`, so adjust the requirements specification accordingly.

Reply:
This thread is very useful. Thanks for creating it, @harupy !